### PR TITLE
tater fix for MobEntityMixin crash

### PR DESCRIPTION
**Pull request by TheFoolKila**
resolves https://github.com/samedifferent/TrickOrTreat/issues/8

### CODE REVIEW

File: src/main/java/samebutdifferent/trickortreat/mixin/MobEntityMixin.java [hunk 1]
```diff
@@ -29,7 +28,8 @@
             method = "<init>"
     )
     private void initMixin(CallbackInfo ci) {
-        if (this instanceof Monster) {
+        MobEntity theMobEntity = (MobEntity) (Object) this;
```
Comment (Jamalam360) [inline src/main/java/samebutdifferent/trickortreat/mixin/MobEntityMixin.java:31]: That's some interesting variable naming but it really doesn't matter ig :help_me:

**Review by Jamalam360 — APPROVED** [review]
I'll get this merged and released in a bit

**Comment by Jamalam360** [comment]
Thanks

File: src/main/java/samebutdifferent/trickortreat/mixin/MobEntityMixin.java (hunk 1 above)
Comment (TheFoolKila) [reply src/main/java/samebutdifferent/trickortreat/mixin/MobEntityMixin.java:31]: :tater_sweat:

**Pull request merged**